Libraries/Opc.Ua.Client/Session.cs

```diff
@@ -1312,7 +1315,8 @@
                 {
                     AddSubscription(subscription);
                 }
-
+                m_lastKeepAliveTime = DateTime.UtcNow;
+                Reconnect();
```

Comment:
really?

Reply:
was left over from testing, will be removed